decidim-proposals/app/forms/decidim/proposals/admin/proposal_form.rb

```diff
@@ -92,6 +117,12 @@ def scope_belongs_to_participatory_space_scope
         def notify_missing_attachment_if_errored
           errors.add(:attachment, :needs_to_be_reattached) if errors.any? && attachment.present?
         end
+
+        def ordered_hashtag_list(string)
+          return [] unless string
+
+          string.split.reject(&:blank?).uniq.sort_by(&:parameterize)
```

Comment:
```suggestion
          string.to_s.split.reject(&:blank?).uniq.sort_by(&:parameterize)
```

This way you don't need the early return